test/unit/codegeneration/SourceMap.js

```diff
@@ -83,12 +82,8 @@ suite('SourceMap.js', function() {
 
     testcases.forEach(function(testcase, caseNumber) {
       var actual = consumer.originalPositionFor(testcase.generated);
-      var shouldBeTrue = actual.line === testcase.original.line;
-      assert.isTrue(shouldBeTrue, caseNumber + ' Line mismatch ' + actual.line);
-      var expected = testcase.original.column;
-      shouldBeTrue = actual.column === expected;
-      assert.isTrue(shouldBeTrue,
-          caseNumber + ' Column mismatch ' + actual.column + ' vs ' + expected);
+      assert.strictEqual(actual.line, testcase.original.line);
```

Comment:
FYI, we usually use `assert.equal` unless we really need strict since it reads better. (I do believe chai made the wrong choice here though.)